src/elements/toggle/toggle-option/toggle-option.visual.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { html, nothing } from 'lit';
+
+import {
+  describeEach,
+  describeViewports,
+  visualDiffDefault,
+  visualRegressionFixture,
+} from '../../core/testing/private.js';
+
+import './toggle-option.js';
+
+describe(`sbb-toggle-option`, () => {
+  let root: HTMLElement;
+
+  const cases = {
```

Comment:
For simple arrays you can directly use an array without a complex object. And then using a for of loop